### PR TITLE
update package metadata

### PR DESCRIPTION
**Pull request by tlugger**
Part of NIO-1128

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,8 +22,9 @@
     tests_require=[
         'requests>=2.3.0'
     ],
-
-    author='n.io',
-    author_email='info@n.io',
-    url='http://n.io'
+	
+    licence='Proprietary',
```
Comment (mattdodge) [inline setup.py]: We should check on this, I think we may want to use a specific license. Unless you've checked with Tim already and this is what we want?

Comment (bp1222) [reply setup.py]: Should probably add the classifiers for this as well.

https://pypi.org/pypi?%3Aaction=list_classifiers

Comment (mattdodge) [reply setup.py]: Confirmed with Tim, this should be `Apache 2.0` license. We can update that in the setup.py and elsewhere in the repo as needed

File: setup.py [hunk 2]
```diff
@@ -22,8 +27,13 @@
     tests_require=[
         'requests>=2.3.0'
     ],
+	
+    classifiers=[
+        'License :: OSI Approved :: Apache Software License',
```
Comment (tlugger) [inline setup.py]: I do want to confirm, is this the correct classifier for `Apache 2.0`? It seems to be the only one that fits from the list @bp1222 commented with.

Comment (tlugger) [reply setup.py]: Although, per [this](https://github.com/pypa/pypi-legacy/issues/91), Apache 2.0 is the only version that is OSI Approved so this should actually be fine

Comment (mattdodge) [reply setup.py]: yeah this looks good to me 👍

**Pull request merged**